app/Ama.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace App;
+
+class Ama 
```

Comment:
The ama class should extend the eloquent model - `Illuminate\Database\Eloquent\Model`. This way it's only a regular class and it's missing the main functionality of the model. Add the extend here (refer to the class with `use` at the top and then do `extends Model`).